documentation/modules/exploit/linux/http/netgear_r7000_cgibin_exec.md

```diff
@@ -0,0 +1,62 @@
+The netgear_r7000_cgibin_exec module exploits a command injection vulnerability in Netgear R7000 and R6400 router firmware version `1.0.7.2_1.1.93` and possibly earlier. The vulnerability is found in the `/cgi-bin/` folder of the router. A manual injection would look like so: `http://<RouterIP>/cgi-bin/;echo$IFS"cowsay"`. This will echo 'cowsay' on the router. 
+
+
+## Vulnerable Application
+
+Netgear R7000 and R6400 routers running firmware version `1.0.7.2_1.1.93` and possibly earlier.
+
+## Verification Steps
+
+  1. Start msfconsole
+  2. Do: `use exploit/linux/http/netgear_r7000_cgibin_exec`
+  3. Do: `set RHOST <RouterIP>`
+  4. Do: `set PAYLOAD <payload>
+  5. Do: `run`
+  6. If the router is a R7000 or R6400, the module should run
+
+## Options
+
+  **RHOST**
+
+  This should usually be the local IP address of the vulnerable router.
+
+  **PAYLOAD**
+  
+  The valid payloads are `cmd/unix` payloads _only_, as this is a command execution module
+
+## Scenarios
+
+  Sample output of the options looks like so 
```

Comment:
do you have one of these vuln routers that you tested it against?  If so, drop this scenario stuff and put that in here instead.  If not, leave it blank and we'll hope someone else has one that can fill this in.